src/php/inc/setup-queries.php

```diff
@@ -10,8 +10,34 @@
  * @return array
  */
 function sb_query_vars_filter( $vars ) {
-	$vars[] = 'example_var'; // e.g. allow ?example_var="test" in url.
+	// Allow custom query params in URL.
+	// e.g. $vars[] = 'example_var'; to allow ?example_var=test in URL.
 	return $vars;
 }
-
 add_filter( 'query_vars', 'sb_query_vars_filter' );
```

Comment:
Since this function isn't doing anything, do we want to comment out the `add_filter` call as well?

Reply:
Good call, updated in https://github.com/sparkbox/sparkpress-wordpress-starter/pull/124/commits/c5077f2a06aba090b7ff0cf795b88635d720dd80